schema/training-module.js

```diff
@@ -25,12 +25,12 @@ class TrainingModule extends BaseModel {
           }
         },
         pass_date: { type: 'string', pattern: date.yearMonthDay },
-        not_applicable: { type: 'boolean' },
-        accrediting_body: { type: 'string' },
-        other_accrediting_body: { type: 'string' },
-        certificate_number: { type: 'string' },
-        exemption: { type: 'boolean' },
-        exemption_description: { type: 'text' },
+        not_applicable: { type: ['boolean', 'null'] },
```

Comment:
Boolean values probably shouldn't be nullable, they should have default values.

Reply:
cool, yeah they do, I just forgot